src/components/footer/footer.css

```diff
@@ -0,0 +1,20 @@
+.margin-bottom {
+  margin-bottom: 64px;
+}
+
+/* stick footer to bottom of page */
+#root {
+  min-height: 100vh;
+  display: flex;
+  flex-direction: column;
+}
```

Comment:
can be moved to `styles.css` if footer being used in all screens

Reply:
I mentioned `styles.css` cuz this class targets an element that's not a part of the footer component. Hence it should ideally be in a global stylesheet.